package.json

```
@@ -12,7 +12,8 @@
         "format": "prettier --write **/*.ts",
         "format-check": "prettier --check **/*.ts",
         "lint": "eslint --ext .ts src/",
-        "all": "yarn type-check && yarn build && yarn format && yarn lint"
+        "all": "yarn type-check && yarn build && yarn format && yarn lint",
+        "test": "node --test --no-warnings=ExperimentalWarning --loader ts-node/esm src/test/*.test.ts"
```

Can we use `tsx` instead? it would be faster and closer to the code we build (as it uses `esbuild`)

```suggestion
        "test": "node --test --no-warnings=ExperimentalWarning --import tsx src/test/*.test.ts"
```

Also, I think `--no-warnings=ExperimentalWarning` can now be removed as the CI now runs on Node v22

---

Also, the github workflow should include this?